os/winapi/save_file.c

```diff
@@ -29,49 +33,57 @@ int open_save_file(const char *path, size_t size,
   LARGE_INTEGER sz;
   int my_created;
 
-  // Open the file, get its size.
-  if ((hfile = CreateFile(path, GENERIC_READ | GENERIC_WRITE,
-    0, NULL, OPEN_EXISTING, FILE_FLAG_RANDOM_ACCESS, NULL))
-    != INVALID_HANDLE_VALUE)
-    my_created = 0;
-
-  else {
+  if (path == NULL) {
+    ptr = calloc(size, 1);
```

Comment:
Slightly pedantic, but please check if return from calloc is NULL and bail accordingly